config/aadpodidentity/aad-pod-identity-deployment.yaml

```diff
@@ -0,0 +1,158 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
@aramase can you please take a look to make sure everything looks good from an AAD pod identity standpoint?

Reply:
Thank you for the ping. I'll take a look at it today.